app/importers/cma_import/attachment_attacher.rb

```diff
@@ -1,8 +1,9 @@
 class CmaImportAttachmentAttacher
   include ::DocumentImport::AttachmentHelpers
 
-  def initialize(create_document_service, assets_directory)
+  def initialize(create_document_service, repo, assets_directory)
```

Comment:
It would be good to be explicit about what kind of repo we're expecting here: `document_repository`